packages/amplify-util-mock/src/CFNParser/appsync-resource-processor.ts

```diff
@@ -257,16 +259,21 @@ export function processResources(
     resources: {},
     exports: {},
   };
-  const processedResources = {
-    schema: {},
+  const processedResources: AmplifyAppSyncSimulatorConfig = {
+    schema: {
+      content: ''
+    },
     resolvers: [],
     functions: [],
     dataSources: [],
     mappingTemplates: [],
     tables: [],
     appSync: {
       name: '',
-      defaultAuthenticationType: {},
+      defaultAuthenticationType: {
```

Comment:
Why OIDC as default auth type? I would prefer this to be APIKey or cognito

Reply:
Changed to API_KEY.